apps/checker/app/services/MatcherPool.scala

```diff
@@ -210,13 +210,14 @@ class MatcherPool(
   private def offerJobToQueue(job: MatcherJob): Future[(MatcherJob, List[RuleMatch])] = {
     logger.info(s"Job has been offered to the queue")(job.toMarker)
 
-    queue.offer(job).collect {
+    queue.offer(job) match {
       case Dropped =>
         failJobWith(job, "Job was dropped from the queue, as the queue is full")
       case QueueClosed =>
         failJobWith(job, s"Job failed because the queue is closed")
       case QueueFailure(err) =>
         failJobWith(job, s"Job failed, reason: ${err.getMessage}")
+      case _ => // Do nothing if job is accepted
```

Comment:
Outside the scope of this PR, but would it be useful to log that the job has been accepted?

Reply:
3782a14f1248608f0da8fed17ea4c84f9a78a31b adds this – it's only little, and it's good to be explicit for every case (what if we were dropping log lines?)